### PR TITLE
CW-136 gitignore and remove fonts.css (auto generated), ...

### PR DESCRIPTION
**Pull request by Semperdecus**
remove blog unique portabletext settings and reuse the default portabletext settings (fixes highlighting, linking, ...), remove double loading="eager" in BlogHighlight.astro, change ComponentWrapper.astro to not have blue bg when bgclass is left empty, use default portabletext settings in highlightblock (fix links and highlighting), update styling for portabletext links, update queries to resolve internallinks, update typings, fix errors, use default portabletext settings in cardgrid highlights and tabs (fix link and highlighting), remove unused code in TeaserWithImage.astro

### CODE REVIEW

File: src/components/cms-components/BlogHighlight/BlogHighlight.astro [hunk 1]
```diff
@@ -50,7 +49,7 @@ const blogPost = Astro.props.blogPost.blogPost;
           {blogPost.intro?.intro && (
             <div class="text-sm font-light">
               <PortableText
-                components={portableBlogHighlightComponents}
+                components={defaultComponents}
```
Comment (DavithkbY) [inline src/components/cms-components/BlogHighlight/BlogHighlight.astro:52]: I think the styles for the bloghighlight are gone. These have different styles that's why I had created custom components for them.

File: src/components/cms-components/HighlightBlock/HighlightBlock.astro [hunk 2]
```diff
@@ -48,16 +48,16 @@ const id = "id_" + Math.random().toString(36).slice(2, 11);
...
-        />
-      )
-    }
-
+    <div class="font-sm prose font-light">
+      {
+        highlight.intro?.intro && (
+          <PortableText
+            value={highlight.intro?.intro}
+            components={defaultComponents}
```
Comment (DavithkbY) [inline src/components/cms-components/HighlightBlock/HighlightBlock.astro:56]: Again this component's portabletext had custom styling.

File: src/components/uielements/BlogPreview.astro [hunk 3]
```diff
@@ -62,7 +62,7 @@ const { blogPost } = Astro.props;
         <div class="text-sm font-light">
           {blogPost.intro?.intro && (
             <PortableText
-              components={portableBlogHighlightComponents}
+              components={defaultComponents}
```
Comment (DavithkbY) [inline src/components/uielements/BlogPreview.astro:65]: this component had custom portable text styling that differs from the default.

File: src/components/uielements/SmallTile.astro [hunk 4]
```diff
@@ -31,7 +28,7 @@ const HeadlineTagName = headlineTag;
       set:html={headline}
     />
     <span class="text-center text-sm font-light">
-      <PortableText value={description} />
+      <PortableText value={description} components={defaultComponents} />
```
Comment (DavithkbY) [inline src/components/uielements/SmallTile.astro:31]: This components portabletext had custom styling.

**Review by DavithkbY — COMMENTED** [review]
Everything looks good except the portable text changes. We need to find a clearer solution to this. But I already spent hours and couldn't find any solution but this.

File: src/components/uielements/SmallTile.astro (hunk 4 above)
Comment (Semperdecus) [reply src/components/uielements/SmallTile.astro:31]: Two reasons: 
- I think text and headings should remain consistent throughout the website. Makes no sense to have a slightly thicker title for blog posts which is hardly noticable Maybe ask a frontender about their opinion as well.. @CosmicEvil @maartjegroenen 

H3 Before:
`<h3 class="text-balance font-serif text-[1.375rem] font-medium sm:text-lg">`

H3 After:
`<h3 class="text-balance font-serif text-xl font-medium sm:text-2xl">`

Even better would be to have a central styling for heading and span tags

- Second reason is that the custom youtube, image and link components don't work with the old portable text settings

Comment (DavithkbY) [reply src/components/uielements/SmallTile.astro:31]: You should discuss this with Manon Wouters. She made the design.

Comment (DavithkbY) [reply src/components/uielements/SmallTile.astro:31]: And it doesn't make sense to have all the headings etc styled the same. Sometimes you want to use a h2 where it is semantically correct to use a h2 but you don't want it to be as huge as a title h2 for example.

Comment (Semperdecus) [reply src/components/uielements/SmallTile.astro:31]: fyi two portable texts have changed from custom to default styling.

1. Blog post intro (which is very different and needs to be reviewed) 
Currently:
image
On dreamy trifle design:
image

2. highlightblock text, which is only different for heading tags which you probably won't use anyway in the regular text.
Currently:
image

Design:
image


so all in all there's just nothing that actually changed

Comment (DavithkbY) [reply src/components/uielements/SmallTile.astro:31]: But the heading tags are now not correct no? It used to be that the heading tags on the cms version were the same as dreamy trifle.

Comment (Semperdecus) [reply src/components/uielements/SmallTile.astro:31]: these changes don't affect the cms version. Not sure if I understand what you mean? Which heading tags were affected by this?

Comment (DavithkbY) [reply src/components/uielements/SmallTile.astro:31]: Ok then this PR can be merged

**Pull request merged**